tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Supplier Section Update.feature

```diff
@@ -116,7 +116,21 @@ Scenario: 6. A buyer user cannot update a supplier section for an organisation t
     Then a response with status code 403 is returned
 
 @4621
-Scenario: 7. Service Failure
+Scenario: 7. A user with read only permissions cannot update a supplier section
+    Given the user is logged in with the Readonly-Buyer role for organisation e6ea864e-ef1b-41aa-a4d5-04fc6fce0933

```

Comment:
Similar thing as the previous comment. Could this test be simplified to not include most of the order information?

Reply:
👍